orchestrator/startup.sh

```diff
@@ -14,7 +14,7 @@ echo orchestrator \
     --ethereum-rpc="$ethrpc" \
     --fees="${DENOM}"
 
-orchestrator \
+gorc \
```

Comment:
The api changed here. I don't know if we still need all these flags. Most of this is in the config file now.

```
gorc --config /path/to/config.toml orchestrator start --ethereum-key signer --cosmos-key orchestrator
```